### PR TITLE
Improved readme.js

### PR DESCRIPTION
**Pull request by Peiprjs**
Made it look a bit better

### CODE REVIEW

**Comment by Peiprjs** [comment]
I think you'll like it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,10 +16,11 @@
 * We are accepting contributions in any language.
 * Use meaningful variable, method and function names and comments.
 * No profanity.
+* Basically, use common sense.
```
Comment (tomassirio) [inline CONTRIBUTING.md:19]: Pls remove this line. We are not patronising  the users

**Review by tomassirio — APPROVED** [review]
Dude. That was THE REAL SHIT

File: CONTRIBUTING.md (hunk 1 above)
Comment (Peiprjs) [reply CONTRIBUTING.md:19]: ok, gimme a sec
```suggestion
* Basically, use common sense.
```

Comment (Peiprjs) [reply CONTRIBUTING.md:19]: Gosh damnit 
```suggestion

```

**Pull request merged**